aptos-move/vm-genesis/src/lib.rs

```diff
@@ -741,17 +741,20 @@ fn initialize_keyless_accounts(session: &mut SessionExt, chain_id: ChainId) {
 }
 
 fn create_accounts(session: &mut SessionExt, accounts: &[AccountBalance]) {
-    let accounts_bytes = bcs::to_bytes(accounts).expect("AccountMaps can be serialized");
-    let mut serialized_values = serialize_values(&vec![MoveValue::Signer(CORE_CODE_ADDRESS)]);
-    serialized_values.push(accounts_bytes);
-    exec_function(
-        session,
-        GENESIS_MODULE_NAME,
-        "create_accounts",
-        vec![],
-        serialized_values,
-    );
-}
+    for account in accounts { // creating accounts one by one to avoid the quadratic complexity of the Move function create_accounts.
+        let accounts = vec![account];
+        let accounts_bytes = bcs::to_bytes(accounts.as_slice()).expect("AccountMaps can be serialized");
```

Comment:
```suggestion
        let accounts_bytes = bcs::to_bytes(accounts.as_slice()).expect("AccountMaps could not be serialized");
```

Reply:
The standard in Rust is to state the expectation instead of the error. I find it confusing, but we've decided to stick with it.

Reply:
I've corrected the message.